### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by RBN2208**

### CODE REVIEW

File: src/components/Landingpage/Landingpage.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+export default function Landingpage({ isLoaded }) {
+  const [hidePage, setHidePage] = useState(false)
+
+  setTimeout(() => {
+    setHidePage(true)
+  }, 6500)
+
+  return (
+    <FullWrapper
```
Comment (VictoriaWika) [inline src/components/Landingpage/Landingpage.js:12]: Maybe you could use another name here, I'm not quite sure what the wrapper is wrapping 😄 Something like Pagelayout?

File: src/components/Landingpage/Landingpage.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+const FullWrapper = styled.div`
+  ${props => (props.hidePage ? 'display:none;' : 'display: flex;')}
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  opacity: 1;
+  z-index: 1;
```
Comment (VictoriaWika) [inline src/components/Landingpage/Landingpage.js:33]: You could use a variable for z-index, at least thats what Jerry suggested in my code 😄

File: src/components/Landingpage/Landingpage.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      <Textbox>caluFix</Textbox>
+      <Loadingbar>
+        <Loader></Loader>
+      </Loadingbar>
+    </FullWrapper>
+  )
+}
+
+const FullWrapper = styled.div`
+  ${props => (props.hidePage ? 'display:none;' : 'display: flex;')}
```
Comment (VictoriaWika) [inline src/components/Landingpage/Landingpage.js:26]: display: ${props => (props.hidePage ? 'none;' : 'flex')};
I think you could write this line like that... That way, you could see that you are changing display from the start.

File: src/components/Landingpage/Landingpage.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  position: absolute;
+  top: 60%;
+  width: 200px;
+  height: 10px;
+  border-radius: 50px;
+  background-color: var(--color-green);
+  overflow: hidden;
+  box-shadow: 0 0 10px white;
+`
+const loadingAnimation = keyframes`
```
Comment (VictoriaWika) [inline src/components/Landingpage/Landingpage.js:62]: Nice that you use keyframes 👍🏻

**Review by VictoriaWika — COMMENTED** [review]
Nice, I left some small suggestions 😄

File: src/components/Landingpage/Landingpage.js (hunk 3 above)
Comment (RBN2208) [reply src/components/Landingpage/Landingpage.js:26]: Thats a good point! Thanks!

**Pull request merged**